storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1574,6 +1577,15 @@ Disable with --skip-innodb-checksums.",
      &opt_rocksdb_wal_dir, &opt_rocksdb_wal_dir, 0, GET_STR_ALLOC, REQUIRED_ARG,
      0, 0, 0, 0, 0, 0},
 
+    {"register_redo_log_consumer", OPT_REGISTER_REDO_LOG_CONSUMER,
+     "Register a redo log consumer in the start of the backup. If this option "
+     "is enabled, it will block the server from purging redo log if PXB redo "
+     "follow thread is still copying it, causing stall in writes on the "
```

Comment:
still copying it and will stall DMLs on the server

Reply:
@altmannmarcelo ^

Reply:
@satya-bodapati  Done